core/runtime/bindings.rs

```diff
@@ -402,7 +402,14 @@ pub(crate) fn initialize_deno_core_ops_bindings<'s>(
 
     for method in op_method_ctx.static_methods.iter() {
       let op_fn = op_ctx_template(scope, method);
-      let method_key = method.decl.name_fast.v8_string(scope).unwrap();
+      let method_key = method.decl.name;
+      let method_key = method_key.strip_prefix("__static_").unwrap();
+      let method_key = v8::String::new_from_one_byte(
+        scope,
+        method_key.as_bytes(),
+        v8::NewStringType::Normal,
+      )
+      .unwrap();
```

Comment:
Updated to only use the renamed name for the rust functions/types, so the op name itself doesn't get renamed and we don't have to do this string manipulation.

So that lets us use a OneByteConst here (and also for setters, which had the same issue)